LanguageFeatures/Patterns/logical_or_A03_t01.dart

```diff
@@ -28,19 +28,19 @@ import "patterns_lib.dart";
 main() {
   Shape shape = Circle(1);
   switch (shape) {
-    case Square(area: var s1) | Circle(area: var s2):
+    case Square(area: var s1) || Circle(area: var s2):
```

Comment:
I know the placement of the "^^....^" marker is ignored for `unspecified` errors, but we might as well keep them consistent: Just a simple `^` right after `//` in order to indicate that the position is ignored, or (the more readable option), a real "^^...^" somewhere on the line which is likely to be the actual location of the error.

I guess it should work to grep the diff for "^" with a context of 1 line to find all these.